.pre-commit-config.yaml

```diff
@@ -11,6 +11,17 @@ repos:
   - id: end-of-file-fixer
   - id: trailing-whitespace
 
+- repo: https://github.com/asottile/pyupgrade
+  rev: v2.37.3
+  hooks:
+    - id: pyupgrade
+      args: ["--py38-plus"]
+
+- repo: https://github.com/fsouza/autoflake8
```

Comment:
I mentioned this in another PR, but here too it sounds like autoflake8 has been abandoned in favor of autoflake.